File: examples/Eq/CLEVER_Eq/yiBound/yiBound_upgr.c
int lib(int x) {
  if (x > 11)
    return 11;
  else
    return x - 1;
}

int client(int x) {
  if (x < -100 || x > 100) {
    return x;
  }
  if (x > lib(x))
    return x;
  else
    return lib(x);
}

int main() {
  int x;
  return client(x);
}

File: examples/eq/oneBound/yiBound_upgr.py
def lib(x):
    if x > 10:
        return 12
    else:
        return x+1

def yiBound(x):
    if x < -100 or x > 100:
        return x
    if x > lib(x):
        return x
    else:
        return lib(x)
